config/gpu_configs/gpu_aquaplanet_dyamond_summer.yml

```diff
@@ -0,0 +1,30 @@
+dt_save_state_to_disk: "Inf"
```

Comment:
This feature is not being tested anywere, we should add it to buildkite and maybe add a couple of plots too